### ISSUE TITLE
verify technic solder migration

### ISSUE THREAD

**Issue by maggi373** [Thorfusion/solder.py#93]

**Comment by Pyker** [comment]
Isn't key verification already implemented in https://github.com/Thorfusion/solder.py/blob/82ed597ef415c5e28c551b3c7e44903e54d5a82e/api.py#L22-L34 ?

**Comment by maggi373** [comment]
> Isn't key verification already implemented in
> 
> https://github.com/Thorfusion/solder.py/blob/82ed597ef415c5e28c551b3c7e44903e54d5a82e/api.py#L22-L34
> 
> ?

yes its already implemented, however we didn't have the "full" implementation on api/modpack and therefore adding new modpacks to technic wasn't working. however this is an badly written bug report by me. currently there are issues doing the database upgrade from technic solder to solder.py. thats this is referencing.

**Comment by maggi373** [comment]
verifired to work
